nautobot/docs/installation/services.md

```diff
@@ -211,6 +253,15 @@ Then, start the `nautobot` and `nautobot-worker` services and enable them to ini
 $ sudo systemctl enable --now nautobot nautobot-worker
 ```
 
+If you are also running the RQ worker, repeat the above command for the RQ service:
+
+```no-highlight
+sudo systemctl enable --now nautobot-rq-worker
+```
+
+!!! tip
+    If you are running the concurrent RQ worker, you must remember to enable/check/restart the `nautobot-rq-worker` process as needed, oftentimes in place of the `nautobot-worker` process. 
```

Comment:
```suggestion
    If you are running the concurrent RQ worker, you must remember to enable/check/restart the `nautobot-rq-worker` process as needed, oftentimes in addition to the `nautobot-worker` process. 
```

Reply:
Thanks for adding this tip! But it's not quite correct since for v1.1.0 and later the `nautobot-worker` (Celery) service must always be running for background services to operate, since the RQ worker will only be used to execute custom tasks that haven't been migrated.

Reply:
not sure I follow @jathanism's comment here, I think we are in agreement.  The idea of the tip is to remind users any time they need to enable/check/restart the nautobot-worker (celery) service they probably also need to perform the same action with the nautobot-rq-worker (rq) service

Reply:
added Nick's suggested edit, which made sense